### ISSUE TITLE
LWT can't be installed on AMO -dev

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-frontend#6986]
STR:

1. Load a LWT details page on AMO -dev - https://addons-dev.allizom.org/en-US/firefox/addon/tiger-eyes-light-blue/?src=rating
2. Try to install the theme

Actual result:
The install button is disabled

Expected result:
Theme can be installed

Notes:
- LWT installations are not working from disco pane either
- not reproduced with other addon types
- reproduced on AMO -dev with FF63, Win10x64 and Android 8.0

![image](https://user-images.githubusercontent.com/31961530/48768136-0f0aa800-ecc1-11e8-8788-b5f373e03e42.png)

**Comment by AlexandraMoga** [comment]
Could be caused by https://github.com/mozilla/addons-frontend/issues/6903 ?

**Comment by willdurand** [comment]
Yep, it is because of a recent change. I think LWT don't have an `installURL`. I am on it!

**Comment by willdurand** [comment]
It is because of this: https://github.com/mozilla/addons-frontend/pull/6980
